features/fixtures/business_readiness.json

```diff
@@ -82,41 +82,31 @@
                   },
                   {
                     "content_id": "94b3cfe2-af89-4744-b8d7-7fc79edcbc85",
-                    "title": "Agriculture",
+                    "title": "Agriculture and forestry (including wholesale)",
                     "schema_name": "facet_value",
                     "details": {
-                      "label": "Agriculture",
+                      "label": "Agriculture and forestry (including wholesale)",
                       "value": "agriculture"
                     },
                     "links": {}
                   },
                   {
                     "content_id": "43d328b3-59ee-4d5d-bda0-7c7d4bb301be",
-                    "title": "Air transport (aviation)",
+                    "title": "Air freight and air passenger services",
                     "schema_name": "facet_value",
                     "details": {
-                      "label": "Air transport (aviation)",
-                      "value": "air-transport-aviation"
+                      "label": "Air freight and air passenger services",
+                      "value": "air-freight-air-passenger-services"
                     },
                     "links": {}
                   },
                   {
-                    "content_id": "5faa1741-fc55-4110-b342-de92f6324118",
-                    "title": "Ancillary services",
+                    "content_id": "cf341686-c886-42c7-bf8e-cf177b6a2100",
```

Comment:
Is this a new content_id?

Reply:
Yes, because arts and culture didn't exist as a facet before